pkg/apisix/apisix.go

```diff
@@ -54,6 +54,8 @@ type Cluster interface {
 	Consumer() Consumer
 	// HealthCheck checks apisix cluster health in realtime.
 	HealthCheck(context.Context) error
+	// Plugin returns a Plugin interface that can operate Plugin resources.
+	Plugin() Plugin
```

Comment:
We may use the unified term, This so-called `Plugin` client actually only returns its schema. We may just use the term `PluginSchema`.

Reply:
I will add a `Schema` client for getting schema of APISIX objects (including plugin, route, upstream, etc.) and the `Plugin` will be only used to query names of plugins.

Reply:
Then the title is confusing.

Reply:
updated